evansmusomi/burger-builder
# Improves Data Security

*What this does*
- Guards routes that need protection when not authenticated
- Displays user specific orders

*How to verify*
- A logged out user should not be redirected to the home page if they try to access the orders page
- A logged in user should be able to access the orders page
- A logged in user should only see orders that belong to them
